docs/basics/userguide/save_load.md

```diff
@@ -0,0 +1,158 @@
+(save_load_page)=
+
+# How to save/load an experiment
+
+
+For this example, we'll use the same code as [ExperimentManager](ExperimentManager_page) (from User Guide), and use the save and load functions.
+
+## how to save an experiment
+
+To save your experiment, you have to train it first (with `fit()`), then you just have to use the `save()` function.
+
+Train the Agent :
+```python
+from rlberry.envs import gym_make
+from rlberry_scool.agents.tabular_rl import QLAgent
+from rlberry.manager import ExperimentManager
+
+from rlberry.seeding import Seeder
+
+seeder = Seeder(123)  # seeder initialization
+
+env_id = "FrozenLake-v1"  # Id of the environment
+env_ctor = gym_make  # constructor for the env
+env_kwargs = dict(
+    id=env_id, is_slippery=False
+)  # give the id of the env inside the kwargs
+
+
+experiment_to_save = ExperimentManager(
+    QLAgent,  # Agent Class
+    (env_ctor, env_kwargs),  # Environment as Tuple(constructor,kwargs)
+    init_kwargs=dict(
+        gamma=0.95, alpha=0.8, exploration_type="epsilon", exploration_rate=0.25
+    ),  # agent args
+    fit_budget=int(300000),  # Budget used to call our agent "fit()"
+    n_fit=1,  # Number of agent instances to fit.
+    seed=seeder,  # to be reproductible
+    agent_name="PPO" + env_id,  # Name of the agent
+    output_dir="./results/",  # where to store the outpus
+)
+
+experiment_to_save.fit()
+print(experiment_to_save.get_agent_instances()[0].Q)  # print the content of the Q-table
+```
+Then save:
+```python
+experiment_to_save.save()  # save the model inside the 'output_dir'
+```
```

Comment:
it is now done automatically no? Maybe instead you can just explain how/where save is done. And also have a small explanation of how to save in temporary folder instead of saving in an actual folder.

Reply:
done !